capitoli/Quinto_Passo_Primi_Programmi.rst

```diff
@@ -49,53 +51,38 @@ Le due scatole conterranno il numero 5.
 
-+--------------------------+-------------------------+
-|**Programma1 PIPPO=PLUTO**| **Il risultato sarà:**  |
-+==========================+=========================+
-| PIPPO=5                  |                         |
-|                          |                         |
-| PLUTO=15                 |                         |
-|                          |                         |
-| print("PIPPO=",PIPPO)    | PIPPO=5                 |
-|                          |                         |
-| print("PLUTO=",PLUTO)    | PLUTO=15                |
-|                          |                         |
-| PIPPO=PLUTO              |                         |
-|                          |                         |
-| print("Dopo l'esecuzione | Dopo l'istruzione       |
-| dell'istruzione          |                         |
-|                          | PLUTO=15                |
-| PIPPO=",PIPPO)           |                         |
-|                          |                         |
-| print("Dopo l'esecuzione | PIPPO=15                |                  
-| dell'istruzione          |                         |
-| PLUTO=",PLUTO)           |                         |         
-|                          |                         |
-+--------------------------+-------------------------+
-|**Programma2 PLUTO=PIPPO**| **Il risultato sarà:**  |
-+--------------------------+-------------------------+
-| PIPPO=5                  |                         |
-|                          |                         |
-| PLUTO=15                 |                         |
-|                          |                         |
-| print("PIPPO=",PIPPO)    | PIPPO=5                 |
-|                          |                         |
-| print("PLUTO=",PLUTO)    | PLUTO=15                |
-|                          |                         |
-| PLUTO=PIPPO              |                         |
-|                          |                         |
-| print("Dopo l'esecuzione | Dopo l'istruzione       |
-| dell'istruzione          |                         |
-|                          | PLUTO=5                 |
-| PLUTO=",PLUTO)           |                         |
-|                          |                         |
-| print("Dopo l'esecuzione | PIPPO=5                 |                  
-| dell'istruzione          |                         |
-| PIPPO=",PIPPO)           |                         |         
-|                          |                         |
-+--------------------------+-------------------------+
-
-**II istruzione**::boltblue:`INCREMENTO` 
+**Programma1 PIPPO=PLUTO**
+::
+
+	PIPPO=5
+	PLUTO=15
+	print ("PIPPO = ", PIPPO)
+	print ("PLUTO = ", PLUTO)
+	PIPPO = PLUTO
+	print ("Dopo l'esecuzione dell'istruzione PIPPO = ", PIPPO) 
+	print ("Dopo l'esecuzione dell'istruzione PLUTO = ", PLUTO)
+	
+	# Risultato Atteso:
+		# PIPPO = 5
+		# PLUTO = 15
+		# Dopo l’esecuzione dell’istruzione PLUTO = 15 
+		# PIPPO = 15
```

Comment:
Questo output è errato. Dovrebbe essere "Dopo l'esecuzione bla bla"

Reply:
mi potresti speigare qiesto errore per favore @libremente

Reply:
Se guardi il codice, la print è:
`print ("Dopo l'esecuzione dell'istruzione PIPPO = ", PIPPO)`
quindi stamperà
`Dopo l'esecuzione dell'istruzione PIPPO =` e il numero, non solo `PIPPO`

Reply:
Continua ad essere errato l'output. Prova ad incollare il codice in un blocco activecode e vedrai il risultato

Reply:
Continuo a non capire l'erroe perche io il codice lo ho copiato e incollato dal file di testo 
Quindi non riesco a capire @libremente